_clients/data-prepper/configure-logstash-data-prepper.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Configure Logstash for Data Prepper
+parent: Data Prepper
+nav_order: 2
+---
+# Configure Logstash for Data Prepper
+You can run Data Prepper with a Logstash configuration.
+
+As mentioned in the [Getting Started]({{site.url}}{{site.baseurl}}/opensearch/clients/data-prepper/getting-started) guide, you'll need to configure Data Prepper with a pipeline using a `pipelines.yaml` file.
+
+Alternatively, if you have a Logstash configuration `logstash.conf` to configure Data Prepper instead of `pipelines.yaml`.
```

Comment:
It looks like there are some missing words here. Should it be "Alternatively, if you have a Logstash configuration, you can use..."?

Reply:
I don't see this change?